lib/connect/api/helpers/lookups.js

```diff
@@ -16,7 +16,18 @@ const lookupAssetByProductIdExternalId = async (client, productId, assetExternal
   return directory.getAssetsByProductIdExternalId(productId, assetExternalId);
 };
 
+const lookupTierByExternalId = async (client, assetExternalId) => {
+  const tier = new Directory(client);
+  const filter = { external_id: assetExternalId };
+  const results = await tier.searchTierAccounts(filter);
+  if (results.length === 1) {
```

Comment:
i think is not right.

here we must check if who triggers uses hub, and if uses hub.id filter also by that since provider may have more than one hub repeating ids, but we do in scope of given hub and account lives on a hub. If there is no hub due other reason then is fine to avoid it.

in fact here we have 2 different cases:

a) order everybody new => well....accounts bound to connection hub
b) Order with some people new, some not:
     - We must search accounts based on a hub if present: If account exists => Reuse if not create new one
     - If no hub present, we can assume that provider has no more than 1 hub, maybe we are lucky expanding